scripts/sample_maps.py

```diff
@@ -133,16 +145,19 @@ def main(_):
       #ke_cluster, kb_cluster = ks93(g1_cluster, g2_cluster)
 
     # Add noise the shear map
-    gamma1 += FLAGS.sigma_gamma * onp.random.randn(map_size,map_size)
-    gamma2 += FLAGS.sigma_gamma * onp.random.randn(map_size,map_size)
+    if FLAGS.cosmos_noise_realisation:
+      print('cosmos noise real')
+      gamma1 += fits.getdata(FLAGS.cosmos_noise_e1).astype('float32')
+      gamma2 += fits.getdata(FLAGS.cosmos_noise_e2).astype('float32')
+
+    else:
+      gaussian_noise = fits.getdata(FLAGS.gaussian_noise).astype('float32')
```

Comment:
hummmm.... ok I see.... ytou could also have set the seed with a the jax.random.normal right?

Reply:
but this should work :-)

Reply:
I agree that using `jax.random.normal` with constant seed seems the best way to do this. It's fixed